app/views/accounts/_personal_details.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-summary-card">
+  <div class="govuk-summary-card__title-wrapper">
+    <h2 class="govuk-summary-card__title">Personal details</h2>
+  </div>
+  <div class="govuk-summary-card__content">
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dd class="govuk-summary-list__value">
+          <%= govuk_link_to("Visit your DfE Identity account", "#{tra_oidc_domain}account?client_id=#{tra_oidc_client_id}&redirect_uri=#{tra_oidc_redirect_uri}&sign_out_uri=#{tra_oidc_redirect_uri}") %> to check or change your personal details.
```

Comment:
That would probably be a better approach. I put the envs in a separate helper, I'll move the links there as well and it would look a lot cleaner/maintainable.